mirror/ordering-key-different.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Defining the Ordering Key in ClickHouse differently from the Primary Key in Postgres"
+---
+
+Deduplication inconsistencies in ReplacingMergeTree could occur when the Ordering Key in ClickHouse differs from the Primary Key in Postgres. In ClickHouse, the Ordering Key defines: a) the skip index and data order on disk, and b) the deduplication key - group of columns on which data de-duplication occurs.
+
+Users might select an Ordering Key that differs from the Primary Key in Postgres to optimize query performance, without realizing that this choice can lead to deduplication inconsistencies in the data. Below are scenarios where deduplication can go wrong:
+
+* **UPDATE on an Ordering Key Column:** For a row, if a column in the Ordering Key is updated, PeerDB replicates the latest version of that row from Postgres, as Postgres logical replication provides only the latest version by default. ClickHouse does not deduplicate this row with the previous version because the Ordering Key value has changed and is distinct from the previous version. This results in an additional row in ClickHouse compared to Postgres.
```

Comment:
May want to mention `PEERDB_CLICKHOUSE_ENABLE_PRIMARY_UPDATE`

Reply:
We should make this default when customers use custom Ordering Key cc @Amogh-Bharadwaj @iskakaushik